formats/travel_advice/publisher/details.json

```diff
@@ -93,6 +111,26 @@
           "type": "string"
         }
       }
+    },
+    "parent": {
+      "type": "object",
+      "additionalPropeties": false,
+      "required": ["parent"],
```

Comment:
@bradwright thanks, my bad, didn't quite get how `oneOf` worked. I've updated to make these required.